cas-server-core-configuration/src/main/java/org/apereo/cas/configuration/model/support/couchbase/ticketregistry/CouchbaseServiceRegistryProperties.java

```diff
@@ -0,0 +1,59 @@
+package org.apereo.cas.configuration.model.support.couchbase.ticketregistry;
+
+/**
+ * This is {@link CouchbaseServiceRegistryProperties}.
+ *
+ * @author Misagh Moayyed
+ * @since 5.0.0
+ */
+
+public class CouchbaseServiceRegistryProperties {
+    private boolean queryEnabled = true;
+
+    private String nodeSet = "localhost:8091";
+
+    private int timeout = 10;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ['10' is a magic number.](https://www.codacy.com/app/mmoayyed/cas/file/2769871190/issues/source?bid=3256115&fileBranchId=3357337#l15)